_posts/2020-11-09-pcr-primer-calculator-using-r.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: post
+title:  "Designing Primers using R"
+date:   2020-11-09 12:54:04 -0700
+categories: PCR primer melting temperature
+---
+Hello and welcome to my blog post about creating your own PCR primer melting temperature calculator using R! You might be asking your self, what is a PCR Primer? Why do I need to know the melting temperature of my PCR primer? Perhaps, you are here simply because you were assigned to read this blog post to determine the quality of its content based on some kind of grading scale. Nevertheless, all your questions and more will be answered here! So, without further ado, lets get into it.
+
+## What is a PCR Primer?
+To a Biologist a primer is a highly sophisticated tool that allows a scientist to conduct several laboratory experiments. This versatile tool is comprised of a single-stranded chain of nucleotides, typically ranging from 15 to 30 bases long. Scientists add additional bases to the single stranded molecule to customize it for their specific polymerase chain reaction (PCR) experiment. For example, a fluorescent molecule can be attached to a PCR primer for use in RT-PCR experiments. While the application of these primers are numerous, we will be focusing on primers designed for PCR cloning.
```

Comment:
You can remove the first 3 words "To a biologist"
Also, I would add a hyphen to "single-stranded molecule", as you've done above.